app/controllers/application_controller.rb

```diff
@@ -2,4 +2,10 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :null_session
+
+  def respond_with_json(data, status)
```

Comment:
Could not we give more use to this method than just `app/controllers/metric_configurations_controller.rb`? What do you think about an issue mapping this? Looks like we can remove lots of duplication.